pkgs/development/python-modules/safe-pysha3/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "safe-pysha3";
+  version = "1.0.3";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kd+lb9t5ney50BvdfbwPKAb0Ro0sKW+DtuN9hlMZF8I=";
+  };
+
+  meta = {
+    description = "SHA-3 (Keccak) for Python 3.9 - 3.11";
```

```suggestion
    changelog = "https://github.com/5afe/pysha3/releases/tag/v${version}";
    description = "SHA-3 (Keccak) for Python 3.9 - 3.11";
```

Add the changelog pattern.